### PR TITLE
Backend/client file upload #71

### PR DESCRIPTION
**Pull request by GD-Dheer**
- Allows User to Upload a JSON file under uploadFile.
- Frontend shows a page to allow update with functional button for API request
- Backend accepts the API request, parses the data and saves it calling correct models

### CODE REVIEW

File: Client/package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
-    "selenium-webdriver": "^4.0.0-alpha.7"
+    "selenium-webdriver": "^4.0.0-alpha.7",
+    "styled-components": "^5.2.1"
```
Comment (silent-po2) [inline Client/package.json:20]: Really glad this fix is included; it had been giving me a headache earlier

File: Client/src/Components/NavigationMenu.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ import HomeView from '../Views/HomeView'
 import MenuIcon from '@material-ui/icons/Menu'
 import React from 'react'
 import clsx from "clsx"
+import FileUploadView from "../Views/FileUploadView"
```
Comment (silent-po2) [inline Client/src/Components/NavigationMenu.js:20]: The file upload page is simple, but functional. Also has a filler image there, which later on can be replaced with something proper and possibly be used as a 'drag and drop' area. Good work!

**Review by silent-po2 — APPROVED** [review]
All in all this is great work. File upload works and some bad errors will be fixed by this as well.

**Review by TiraFire — APPROVED** [review]
Tested, functions, uploaded a file and it stores locally. Great job GD and Sean

**Pull request merged**